support/azure/azure-kubernetes/create-upgrade-delete/error-code-vhdfilenotfound.md

```diff
@@ -30,20 +30,26 @@ Under rare circumstances, the 124 exit code for the Azure Virtual Machine Scale
 | `K8SAPIServerConnFailVMExtensionError`      | 51                |
 | `K8SAPIServerDNSLookupFailVMExtensionError` | 52                |
 
-It usually hides a network connectivity issue from the AKS node to `mcr.microsoft.com`, or to the API server's fully qualified domain name (FQDN) address.
+This error occurs when there’s a connectivity issue between your AKS cluster and the required Azure endpoints, such as `mcr.microsoft.com` or `acs-mirror.azureedge.net`.
 
 ## Solution
 
-Follow the troubleshooting guides in the following articles:
+Review [outbound network and FQDN rules for Azure Kubernetes Service (AKS) clusters](/azure/aks/outbound-rules-control-egress) and make sure that all API services FQDN are all allowed.
+
+For detailed troubleshooting steps, refer to the troubleshooting guides in these articles
```

Comment:
For detailed troubleshooting steps, refer to the troubleshooting guides in the following articles: